src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -39,6 +39,8 @@ FixedwingPositionControl *l1_control::g_control;
 static int _control_task = -1;			///< task handle for sensor task */
 
 FixedwingPositionControl::FixedwingPositionControl() :
+	_sub_airspeed(ORB_ID(airspeed), 0, 0, nullptr),
```

Comment:
The actual airspeed is better to use if available (control loops are tuned to work with it). Estimate airspeed would only be used if an airspeed sensor was not fitted or was faulty. For airspeed estimation without a sensor, the sideslip fusion in the EKF needs to be enabled via the EKF2_AID_MASK parameter.

Investigating this capability and documenting it's performance and limitations can be done as a follow-up piece of work.